Example/TABTestKit_ExampleUITests/TableSelectionScreen.swift

```diff
@@ -14,6 +14,5 @@ struct TableSelectionScreen: Screen {
     
     let trait = View(id: "TableSelection")
     let navBar = NavBar()
-    let backButton = Button(id: "Table")
-    
+    let backButton = NavBar().backButton(titled: "Table")
```

Comment:
❤️

Reply:
Should this be `navBar.backButton` instead?

```suggestion
    let backButton = navBar.backButton(titled: "Table")
```